ndt7/spec/spec.go

```diff
@@ -28,8 +28,19 @@ const MaxScaledMessageSize = 1 << 20
 // in the appendix of the ndt7 specification.
 const ScalingFraction = 16
 
-// MinMeasurementInterval is the minimum interval between measurements.
-const MinMeasurementInterval = 250 * time.Millisecond
+// AveragePoissonSamplingInterval is the average of a lambda distribution
+// used to decide when to perform next measurement.
+const AveragePoissonSamplingInterval = 250 * time.Millisecond
+
+// MinPoissonSamplingInterval is the min acceptable time that we want
+// the lambda distribution to return. Smaller values will be clamped
+// to be this value instead.
+const MinPoissonSamplingInterval = 25 * time.Millisecond
```

Comment:
You don't have to set min and max. Why are you choosing to do so here?

Reply:
I'm more worried by not choosing max than by not choosing min. The lower bound is set because we don't want to send too close messages. The upper bound because we want ~1 message per second to update the UI.

Reply:
(Incidentally, those limits are the ones recommended by the docs. I can inflate the upper bound as much as one second, if you wish. But I'd like to try to generate ~1 message per second in the worst case. Both for the client's sake and for the data saving sake [*]).

[*] What is the frequency of sampling of the sidecar service? Do we want PASTA also there?

Reply:
Thanks!  All I wanted was a reason. "Because 1 per second is required for good UX" is a great reason.

the tcp_info sidecar service samples at max rate. Whether it should be poisson instead is an interesting question.